Client/src/Utils/CategoryUtils.tsx

```diff
@@ -96,7 +97,7 @@ export function getDisplayName(node: CategoryTreeNode) {
 }
 
 export function getDescription(node: CategoryTreeNode) {
-  return isIndividual(node) ? node.wdkReference.help
+  return isIndividual(node) ? node.wdkReference.description
```

Comment:
Any idea why we were returning `help` here? Was there a good reason for it?

Reply:
My guess is that it has to do with the current implementation `getTooltipContent` - in that function, we seem to be delegating to `getDescription` when the node either (1) isn't an individual or (2) doesn't have a "search" property.

Reply:
Ah, right. I think it also has to do with the fact that `AttributeField` doesn't have a `description` property. I think we might need to have some branching logic based on the type of `wdkReference`.

Reply:
yes the help is for attributes

Reply:
summary for questions.... for questions we would not want to deal with description...